packages/storybook-builder-vite/codegen-importfn-script.js

```diff
@@ -46,14 +46,48 @@ async function toImportFn(stories) {
 
 module.exports.generateImportFnScriptCode =
     async function generateImportFnScriptCode(options) {
+        let normalizeStories;
+
+        // Try to import normalizeStories from @storybook/core-common >= v6.4.
+        try {
+            normalizeStories =
+                require('@storybook/core-common').normalizeStories;
+        } catch (e) {}
+
+        // If it fails, just convert strings to StoriesSpecifier objects.
```

Comment:
My suggestion is to not do any failure handling - just make it crash. We can publish a version 0.2.0 of the builder, which only supports Storybook 6.4+. The try-catch-code here isn't too complex, but I fear things like this will build up complexity and create more issues down the road.

Reply:
I think that makes sense. I didn't want my comments to change the actual implementation. Happy to draw a line and say we only support 6.4+

Reply:
OK! I just reverted the commit that added the failure handling. https://github.com/eirslett/storybook-builder-vite/pull/182/commits/9bec8552967fe9fd3653098e67b4badeaa3ec3b7